core/templates/dev/head/pages/skill_editor/SkillEditorNavbarBreadcrumbDirective.js

```diff
@@ -22,7 +22,18 @@ oppia.directive('skillEditorNavbarBreadcrumb', [
       scope: {},
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/skill_editor/skill_editor_navbar_breadcrumb_directive.html'),
-      controller: [function() {}
+      controller: [
+        '$scope', 'SkillEditorStateService',
+        function($scope, SkillEditorStateService) {
+          var skill = SkillEditorStateService.getSkill();
+          $scope.getTruncatedDescription = function() {
+            var truncatedDescription = skill.getDescription().substr(0, 15);
+            if (skill.getDescription().length > 15) {
```

Comment:
Did you want this to be 40?

Reply:
Yeah, changed it.

Reply:
Yes, I think that's fine for now. We might increase the actual length later. Thanks!